examples/great_expectations_data_validation/README.md

```diff
@@ -0,0 +1,120 @@
+# 🏎 Validate your data with Great Expectations
+In data-centric machine learning development, data quality is critical not only
+to achieve good initial results but also to keep data drift and concept drift
+at bay as your models are deployed to production and interact with live data.
+
+Data validation tools can be employed early on in your machine learning
+pipelines to generate data statistical profiles and infer validation rules
+that can be used to continuously validate the data being ingested at various
+points in the pipeline. For example, data validation rules can be inferred from
+the training dataset and then used to validate the datasets used to perform
+batch predictions. This is one good way of detecting training-serving skew.
+
+## 🗺 Overview
+This example uses the very popular [`Great Expectations`](https://greatexpectations.io/)
+open-source library to run data quality tasks on [the Steel Plates Faults Data Set](https://www.openml.org/search?type=data&sort=runs&id=1504&status=active).
+provided by Semeion, Research Center of Sciences of Communication, Via Sersale
+117, 00128, Rome, Italy. The dataset consists of 27 features describing each
```

Comment:
this could maybe be mentioned in a foot note or similar instead

Reply:
I think it was @strickvl's suggestion that I put it here. I'm comfortable with where it is now.